lib/opal/paths.rb

```diff
@@ -42,6 +42,8 @@ def use_gem(gem_name, include_dependencies = true)
     def require_paths_for_gem(gem_name, include_dependencies)
       paths = []
       spec = Gem::Specification.find_by_name(gem_name)
+      # If spec is nil, something should be raised instead of NoMethodError
+      # in next line
```

Comment:
No, though about using it, but the meaning is different, so if anyone wants to rescue it they can differentiate between the two.